libs/settings/pxt.json

```diff
@@ -0,0 +1,3 @@
+{
+    "additionalFilePath": "../../node_modules/pxt-common-packages/libs/settings"
```

Comment:
I would just have `libs/settings/pxt.json` pointing to `settings---files` with the name overridden to be `settings`

Reply:
will fix